data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -58,7 +58,7 @@ if [ ! -f cvo-bootstrap.done ]
 then
 	echo "Rendering Cluster Version Operator Manifests..."
 
-	rm -rf cvo-bootstrap
+	rm --recursive --force cvo-bootstrap
```

Comment:
Nit: [POSIX `rm`][1] (and most other POSIX commands) only requires the short forms.  Probably doesn't matter, since the particular `rm` implementation we use has non-POSIX long-forms that are unlikely to change, but it does make this script a bit less portable.

[1]: https://pubs.opengroup.org/onlinepubs/9699919799/utilities/rm.html

Reply:
The script already wasn’t portable because there’s a there’s other examples in here of using long flags. For example, mkdir. 

Having a sysadmin background I find the whole use of these long flags on Unix commands less readable and not quickly understood (I didn’t even know -p stood for “parents”), but this was some feedback on another PR and at least the script is consistent I guess.